vendor/knative.dev/pkg/client/injection/kube/informers/core/v1/configmap/fake/fake.go

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Where did these `vendor` changes come from?  I see `vendor/modules.txt` updated but I have to confess I don't really understand how that plays with `go.mod` / `go.sum`.  Did you manually update that file?  Or was it somehow automatically updated?

Reply:
Thank you for the review. The changes in `vendor/modules.txt` are made by the `hack/update-deps.sh` script.